interface1.js

```diff
@@ -722,7 +722,7 @@ window.addEventListener('load', function(){
 			if(autoInputOptions.recursiveCategory === 'x0' || autoInputOptions.noUnary){
 				autoInputOptions.noAdjacentHeads = false;
 			}
```

Comment:
As far as I can tell, you're not adding the information from noBarLevels to autoInputOptions here.